tmail-backend/integration-tests/jmap/memory-jmap-integration-tests/src/test/java/com/linagora/tmail/james/MemoryTeamMailboxMemberSetMethodTest.java

```diff
@@ -33,6 +34,7 @@
 import com.linagora.tmail.module.LinagoraTestJMAPServerModule;
 import com.linagora.tmail.team.TeamMailboxProbe;
 
+@Disabled("Temporarily disable due to unstable test: updateShouldReturnNotUpdatedWhenRemovingOtherManager. Other tests could be unstable as well due to same root cause")
```

Comment:
cf https://github.com/linagora/tmail-backend/pull/1547#pullrequestreview-2622691424

Reply:
Can we please remove this? I'm ok with the rest